Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererArrayBuilder.cs

```diff
@@ -63,14 +64,28 @@ public BocColumnRenderer[] CreateColumnRenderers ()
 
       PrepareSorting(sortingDirections, sortingOrder);
 
-      var firstValueColumnRendered = false;
+      var renderedRowIcon = false;
       var bocColumnRenderers = new List<BocColumnRenderer>(_columnDefinitions.Count);
       var visibleColumnIndex = GetInitialVisibleColumnIndex();
+
+      var enableIconOnFirstPossibleColumn = !_columnDefinitions.Any(cd => cd is IBocColumnDefinitionWithRowIconSupport { RowIconMode: RowIconMode.Preferred });
```

Comment:
Question: my order-by suggestion, did it not work or is it not as readable/understandable as the present code? Am asking to understand how my suggestion helps/doesn't help.

Reply:
NOTE: I just noticed the comments above. Sorry! Need to read.